test/api/getLedger/index.ts

```diff
@@ -0,0 +1,78 @@
+import assert from 'assert-diff'
+import { assertResult, TestSuite } from '../utils'
+import responses from '../../fixtures/responses'
+const { getLedger: RESPONSE_FIXTURES } = responses
+
+export default {
```

Comment:
It wasn't immediately obvious what this file is or where `api` was getting passed in from. Do you think it would be clearer to add a link to the TestRunner here?

Reply:
For sure, some better docs/comments here would make sense.